nms/app/packages/magmalte/app/views/subscriber/SubscriberOverview.js

```diff
@@ -139,28 +145,70 @@ function SubscriberDashboardInternal() {
   );
 }
 
+type Props = {
+  open: boolean,
+  onClose?: () => void,
+  imsi: string,
+};
+
+function JsonDialog(props: Props) {
+  const ctx = useContext(SubscriberContext);
+  const sessionState = ctx.sessionState[props.imsi] || {};
+  const subscriberConfig = ctx.state[props.imsi];
+  const subscriber: mutable_subscriber = {
+    ...subscriberConfig,
+    state: sessionState,
+  };
+  return (
+    <Dialog open={props.open} onClose={props.onClose} fullWidth={true}>
+      <DialogTitle>{props.imsi}</DialogTitle>
+      <DialogContent>
+        <pre>{JSON.stringify(subscriber, null, 2)}</pre>
```

Comment:
minor nit: Perhaps instead of it being displayed as is, can you use the ReactJSON component instead, it might look prettier.

Reply:
Yes it does look prettier !